extensions/vscode/src/api/resources/ContentRecords.ts

```diff
@@ -104,12 +104,18 @@ export class ContentRecords {
   // 204 - no content
   // 404 - contentRecord or config file not found
   // 500 - internal server error
-  patch(deploymentName: string, configName: string, dir: string) {
+  patch(
+    deploymentName: string,
+    dir: string,
+    configName?: string,
+    guid?: string,
```

Comment:
Usability suggestion we could make this a `body` with optional attributes which would mean we wouldn't need to pass `undefined` when sending the `guid` for example.

Reply:
Great recommendation. I've updated the third parameter to now be a body structure with optional attributes and retrofit the calling code. I like it much better.